.github/workflows/update_screens.yml

```
@@ -33,8 +33,12 @@ jobs:
           node-version: 18
           cache: 'yarn'
       - run: yarn install --frozen-lockfile --ignore-scripts
+
+      - name: Switching APT Repositories
+        run: curl --connect-timeout 1 azure.archive.ubuntu.com || (sudo sed -i 's/azure\.//' /etc/apt/sources.list && sudo apt-get update)
```

```suggestion
      - name: [Troubleshooting for Azure VM] Install APT Repositories
        # Связаная проблема https://github.com/actions/runner-images/issues/675
        run: curl --connect-timeout 1 azure.archive.ubuntu.com || (sudo sed -i 's/azure\.//' /etc/apt/sources.list && sudo apt-get update)
```

чтобы было понятно что к чему